kits.md

```diff
@@ -0,0 +1,72 @@
+# Kits
```

Comment:
This is for internal use only, correct? I think we should label it accordingly. Also, is there a way  to limit this so that Ulta won't be able to create kits?